services/QuillLMS/app/services/classlink_integration/districts_importer.rb

```diff
@@ -27,30 +30,34 @@ def run
       end
     end
 
-    def find_or_create_district_and_instance(district_hash)
+    def valid_district?(district_response)
+      in_allowlist?(district_response['tenant_id']) &&
```

Comment:
You are using these string keys for logic in multiple places, so they should be constants.